sdk/communication/azure-communication-phonenumbers/samples/list_acquired_phone_numbers_sample.py

```diff
@@ -7,33 +7,28 @@
 # --------------------------------------------------------------------------
 
 """
-FILE: phone_number_supported_countries_sample.py
+FILE: phone_number_area_codes_sample.py
 DESCRIPTION:
-    This sample demonstrates how to get supported countries via a connection string
+    This sample demonstrates how to get all area codes via a connection string, country code and phone plan id.
 USAGE:
-    python phone_number_supported_countries_sample.py
+    python list_acquired_phone_numbers_sample.py
     Set the environment variables with your own values before running the sample:
     1) AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING - The endpoint of your Azure Communication Service
 """
 
-
 import os
 from azure.communication.phonenumbers import (
-    PhoneNumbersAdministrationClient
+    PhoneNumbersClient
 )
 
 connection_str = os.getenv('AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING')
-phone_number_administration_client = PhoneNumbersAdministrationClient.from_connection_string(connection_str)
-
+phone_numbers_client = PhoneNumbersClient.from_connection_string(connection_str)
 
-def list_all_supported_countries():
-    # [START list_all_supported_countries]
-    supported_countries = phone_number_administration_client.list_all_supported_countries()
-    # [END list_all_supported_countries]
-    print('supported_countries:')
-    for supported_country in supported_countries:
-        print(supported_country)
+def list_acquired_phone_numbers():
+    acquired_phone_numbers = phone_numbers_client.list_acquired_phone_numbers()
+    print('Acquired phone numbers:')
+    print(acquired_phone_numbers)
```

Comment:
This wont actually work - it will print out an `ItemPaged` object reference.
You would need to do something like:
```
print('Acquired phone numbers:')
for number in phone_numbers_client.list_acquired_phone_numbers():
    print(number)
```